### ISSUE TITLE
[Bug] Sorting by Backup Date is opposite of expected order

### ISSUE THREAD

**Issue by MrEngineerMind** [NeoApplications/Neo-Backup#700]
**Description**
If I configure the main app "filter" to sort the app list by backup date, descending, it seems to sort it ascending by backup date

**Steps To Reproduce**

1. Tap Filter button
2. Select "sort by: Backup Date"
3. Select "Descending"
4. Select "User Apps, all backup filters, "all apps" special filter
5. Click "Apply"

**Expected behavior**
The word "Ascending" means "to go higher". In every program I have ever used, when you sort a list in "ascending" order, the first entry in the list will contain the "lowest" value and as you progress through the list, the values "Go Higher" - meaning they "Ascend" i value.

The word "Descending" means "to go lower". In every program I have ever used, when you sort a list in "Descending" order, the first entry in the list will contain the "Highest" value and as you progress through the list, the values "Go Lower". meaning they "Descend" in value.
 
However, the sort order of this app is opposite of that norm - I actually have to select "Ascending" to have the most recent backup at the top of the list.

**Comment by hg42** [comment]
I agree, but I am quite happy that it's wrong :-) it's the sort order I usually want

I think we should rename it to **backup age**?
